export_kakuma_advert_contacts.py

```diff
@@ -24,18 +24,20 @@
                              "credentials bucket")
     parser.add_argument("pipeline_configuration_file_path", metavar="pipeline-configuration-file",
                         help="Path to WUSC-KEEP-II-KAKUMA pipeline configuration json file")
-    parser.add_argument("messages_json_input_path", metavar="messages_json_input_path",
-                        help="Path to the WUSC-KEEP-II-KAKUMA messages traced data JSONL file to extract phone"
-                             "numbers from")
-    parser.add_argument("contacts_csv_path", metavar="contacts-csv-path",
-                        help="CSV file path to write the contacts data to")
+    parser.add_argument("data_dir", metavar="data-dir",
+                        help="Directory path to read messages traced data JSONL file + listening group CSV files "
+                             "to extract phone from, and write the advert CSV file to")
+    parser.add_argument("sms_ad_flow_name", metavar="sms-ad-flow-name",
```

Comment:
Why did this need to change from a path to a flow name?

Reply:
I have refactored the code to read from the main data directory and branch out to the individual directories where necessary. The sms_ad_flow_name is therefore used to define the name of the advert CSV file.

Reply:
Hmm this breaks with convention used elsewhere in the pipeline. The rest of the pipeline's python scripts are very flexible and generally take paths to specific files. The run scripts just provide sensible defaults, including setting the data/Outputs structure. 

Flexible python scripts with sensible defaults in the run scripts still feels like the best structure to me, but as the reason for that is mostly historic I'm happy to talk about re-engineering all the scripts if necessary.

Reply:
In any case, I'm not sure the data directory is the best place to export the advert contacts to. This isn't running automatically as part of the pipeline, we need to be careful not to include phone numbers in the data archives, and TextIt contains the canonical groups anyway.